package.json

```diff
@@ -10,17 +10,33 @@
     "test": "plasmo test"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.3.2",
```

Comment:
@iamstarcode what does this do?

Reply:
Form validations, I thought I removed it, I wanted to add it then realised maybe it isn't important